debian/control

```diff
@@ -82,7 +82,9 @@ Description: Docker container engine - Vim highlighting syntax files
 
 Package: golang-github-docker-docker-dev
 Architecture: all
-Depends: golang-github-docker-docker-credential-helpers-dev, ${misc:Depends}
+Depends: golang-github-ishidawataru-sctp-dev,
+         golang-github-docker-docker-credential-helpers-dev,
+         ${misc:Depends}
 Replaces: golang-docker-dev (<< 1.8.2~ds1-1~)
 Breaks: golang-docker-dev (<< 1.8.2~ds1-1~)
 Provides: golang-docker-dev
```

Comment:
Should this also `Provides: golang-github-docker-libnetwork-dev`, or do you think that's superfluous/overkill?  I guess we don't do anything like that for `github.com/docker/cli`.

Reply:
Fair enough, let's skip it :smile: